CONTRIBUTING.md

```diff
@@ -24,14 +24,14 @@ Thank you for considering to contribute to the go-spacemesh open source project.
 - Add a comment in the issue that you are starting to work on it.
 - Ask for any information that you feel is missing from the issue for you to be able to work on it.
 - Clone branch `Develop` and not `Master` - we follow [gitflow](https://datasift.github.io/gitflow/IntroducingGitFlow.html).
-- Create a new PR (Pull Request) from your cloned and reference the issue number in it.
+- Create a new PR (Pull Request) from your clone and reference the issue number in it.
 - Submit your PR when you believe your code is ready for code review.
 - Follow up on code review comments by one of the project maintainers on your Pull Request (PR)
 - You may add your name and email to AUTHORS when submitting your Pull Request.
 
 # Creating a new issue
 - Scan both open and closed issues and verify that your new feature idea, improvement proposal or bug report is not already addressed in another issue.
-- Before starting to work on a large contribution please chat with the core dev team on our Gitter channel to get some initial feedback prior to doing lots of work.
+- Before starting to work on a large contribution please chat with the core dev team in the #dev_discussion channel on our [discord server](https://chat.spacemesh.io/) to get some initial feedback prior to doing lots of work.
```

Comment:
I think denoting `#dev_discussion` would make it look a bit better. What do you think?

```suggestion
- Before starting to work on a large contribution please chat with the core dev team in the `#dev_discussion` channel on our [discord server](https://chat.spacemesh.io/) to get some initial feedback prior to doing lots of work.
```